Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/DebugPanelErrorIndicator.tsx

```diff
@@ -9,9 +9,9 @@ import { Indicator } from './Indicator';
 
 export const DebugPanelErrorIndicator = (props: { hasError: boolean; hasWarning?: boolean }) => {
   const indicator = props.hasError ? (
-    <Indicator color={`${SharedColors.red10}`} size={5} />
+    <Indicator color={`${SharedColors.red10}`} size={8} />
   ) : props.hasWarning ? (
-    <Indicator color={`${SharedColors.yellow10}`} size={5} />
+    <Indicator color={`${SharedColors.yellow10}`} size={8} />
```

Comment:
this might be only me, but with this color, I can't quite see the boundary of the icon, it's kind of splashed to the behind container. @sangwoohaan what do you think?

Reply:
This warning indicator is actually removed. There is a bigger thread i started regarding all our warning icons across the app not meeting 4.51:1 color contrast ratios. We have this on our agenda for Authoring standup next week when @vivekkshankar is back too and we can have a discussion as a whole.